lib/manager/npm/post-update/yarn.spec.ts

```diff
@@ -45,13 +40,14 @@ describe('manager/npm/post-update/yarn', () => {
   ])(
     'generates lock files using yarn v%s',
     async (yarnVersion, yarnCompatibility, expectedFsCalls) => {
-      Fixtures.mock(
-        {
-          '.yarnrc': 'yarn-path ./.yarn/cli.js\n',
-          'yarn.lock': 'package-lock-contents',
-        },
-        '/some-dir'
-      );
+      fs.readLocalFile.mockImplementation((file) => {
```

Comment:
Revert and use fixtures

Reply:
Why you resolved my comment without any changes or comments?

Reply:
Sorry, it was done by mistake. I probably copied and pasted wrong code.